.github/workflows/labeler-module.yml

```
@@ -0,0 +1,16 @@
+name: "Pull Request Labeler"
+on:
+  pull_request_target:
+    types: [opened]
```

Hi @tupui,

The worflow currently runs after each PRs commit. I think It's possible to change its definition to only have it run when PRs are created, as follows:
```suggestion
name: "Pull Request Labeler"
on:
  pull_request_target:
    types: [created]
```

---

That could work then!

I suggest sending an email to the list as we had this and removed it. You could reuse the old thread we had on the topic.

As I said, me I am +1

---

I am searching for the thread in [SciPy-Dev mailing list archive](https://mail.python.org/archives/list/scipy-dev@python.org/) but I have not been able to identify it. Do you have a pointer?

---

Indeed me neither. It was probably just some exchanges during a meetup and on the PRs to add and remove them. Then maybe post a new message.

(We don't have a proper decision log.)

---

Is posting a message redirecting to this PR in one of the issue you mentioned in https://github.com/scipy/scipy/pull/16870#discussion_r951465958 appropriate?

---

Fine with me if you ping all people that were previously involved. We can do this here directly. I suggest that in your message you remind us all of the previous issue and how this PR solves that. Thanks for doing this!

---

Committing suggestion based on the pieces of feedback given in https://github.com/scipy/scipy/pull/16870#issuecomment-1225679795.